### DISCUSSION TITLE
The Signature modal is incredibly annoying on desktop.

### DISCUSSION THREAD

**Discussion by The-Real-Thisas** [docusealco/docuseal#6] [Ideas]
<img width="1440" alt="Screenshot 2023-07-16 at 5 42 13 PM" src="https://github.com/docusealco/docuseal/assets/59698123/ac86441f-c63e-4330-bfcf-f16be9377395">

If you look at the form on desktop the modal covers almost half of the view which is very backwards for user experience because ideally as a user, they'd more want to read the document they are signing *before* filling it up. 

I see two ways this can be approached. 
- On desktop the document can be on one side (like 3/4 of the page) and then the form inputs on the other side. I kinda made a design similar to this which I'll attach. This way the mobile version can remain as it is but for desktop it can be more useable. 
![image](https://github.com/docusealco/docuseal/assets/59698123/ef8985c7-0c58-4537-aa34-37363a27cb7a)
![image](https://github.com/docusealco/docuseal/assets/59698123/3fb12d8b-8301-4815-847b-a6d0767bab2d)
- You can page them be like interactive overlay over the pdf like in the editor which will pop up the modal only when it is clicked. This would help clean up the mobile version as well.

**Comment by AlexBTurchyn** [comment]
Hi @The-Real-Thisas, I just released a new version with a fix for this issue. The latest version includes a collapse button so now users can close the form to review the entire document and expand it when they are ready to fill out the form/document.

Thanks for your suggestions regarding the layout changes, but I think we should stick to the current approach with a step-by-step form. Hopefully, the "collapse" button will solve this problem for users.

**Reply by The-Real-Thisas** [reply]
I suppose thats an alright compromise. PS like the new text and picture options for the signature.
